### PR TITLE
WHFHRI-694 2.x

### PR DESCRIPTION
**Pull request by David-N-Perkins**
* Updated Dockle action
* Added vulnerability exceptions
* Minor security modifications to the Docker build

## PR Requirements:
 - [ ] The code must be reviewed by at least one Maintainer
 - [x] JIRA Ticket: https://jira.wh-sdlc.watson-health.ibm.com/browse/WHFHRI-694
 - [x] 90 % test code coverage

### CODE REVIEW

File: .github/workflows/ci-workflow.yml [hunk 1]
```diff
@@ -130,12 +130,16 @@ jobs:
...
+        uses: erzz/dockle-action@v1.1.1
         with:
           image: "${{ env.IMAGE_ID }}"
+          report-format: sarif
           exit-code: 1
-          exit-level: 'WARN'
+          failure-threshold: 'WARN'
+          # When updating, also update release.yml
+          accept-filenames: 'config.json'
+          accept-extensions: 'key,p12'
```
Comment (David-N-Perkins) [inline .github/workflows/ci-workflow.yml:142]: I checked the flagged files. There were all due to extra libraries that were installed.

File: .github/workflows/ci-workflow.yml [hunk 2]
```diff
@@ -146,6 +150,8 @@ jobs:
           ignore-unfixed: true
           vuln-type: 'os,library'
           severity: 'CRITICAL,HIGH,MEDIUM'
+          # When updating, also update release.yml
+          skip-dirs: 'home/hri/.bluemix,root/.bluemix,usr/local/ibmcloud/bin/ibmcloud'
```
Comment (David-N-Perkins) [inline .github/workflows/ci-workflow.yml:154]: The IBM CLI plugins had several vulnerabilities, but there's nothing we can do about those. They also aren't part of the deployed code, so we can safely ignore them.

**Review by alisakotliarova — APPROVED** [review]

**Review by aramcodz — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -56,6 +60,8 @@ jobs:
           ignore-unfixed: true
           vuln-type: 'os,library'
           severity: 'CRITICAL,HIGH,MEDIUM'
+          # When updating, also update release.yml
```
Comment (aramcodz) [inline .github/workflows/release.yml:63]: Remove the unnecessary comment here, line 63.

**Review by aramcodz — APPROVED** [review]
One minor comment

File: .github/workflows/release.yml (hunk 3 above)
Comment (David-N-Perkins) [reply .github/workflows/release.yml]: Removed.

**Pull request merged**